### PR TITLE
Remove download estimates

### LINKED ISSUE #470: Download Size Estimates Are Off
screen shot 2018-12-18 at 10 17 49 am

### PR DESCRIPTION
**Pull request by arielsvn**
## Issue Number

close #470 

## Purpose/Implementation Notes

Removes all download size estimates and uses the actual dataset size next to the download link, included with https://github.com/AlexsLemonade/refinebio/pull/979

@dvenprasad do you think the Download File Summary looks good like this?

## Types of changes

* [x] New feature (non-breaking change which adds functionality)

## Functional tests

Download a dataset

## Checklist

* [x] Lint and unit tests pass locally with my changes
* [ ] I have added tests that prove my fix is effective or that my feature works
* [ ] I have added necessary documentation (if appropriate)
* [ ] Any dependent changes have been merged and published in downstream modules

## Screenshots

image

### CODE REVIEW

**Comment by arielsvn** [comment]
Will add here improvements for the designs of the download file summary.

**Comment by dvenprasad** [comment]
screen shot 2019-01-17 at 3 13 00 pm

**Comment by arielsvn** [comment]
Updated file summary:
 
image

File: src/common/helpers.js [hunk 1]
```diff
@@ -276,3 +276,14 @@ export function accumulateByKeys(array, keys) {
     }, {})
   }));
 }
+
+// thanks to https://stackoverflow.com/a/18650828/763705
+export function formatBytes(bytes, decimals = 2) {
```
Comment (dongbohu) [inline src/common/helpers.js]: Interesting approach.

**Pull request merged**